crates/router/src/services/api/client.rs

```diff
@@ -22,7 +22,11 @@ fn get_client_builder(
     proxy_config: &Proxy,
     should_bypass_proxy: bool,
 ) -> CustomResult<reqwest::ClientBuilder, ApiClientError> {
-    let mut client_builder = reqwest::Client::builder().redirect(reqwest::redirect::Policy::none());
+    let mut client_builder = reqwest::Client::builder()
+        .redirect(reqwest::redirect::Policy::none())
+        .pool_idle_timeout(std::time::Duration::from_secs(
+            proxy_config.idle_pool_connection_timeout.unwrap_or(90),
```

Comment:
Can we instead use the `Default` trait implementation for this purpose? Using a number with `unwrap_or()` can be harder to track defaults like this.

Reply:
sure